### DISCUSSION TITLE
How to preselect first item in completion popup menu?

### DISCUSSION THREAD

**Discussion by xzbdmw** [helix-editor/helix#9511] [General]
search for a while in repo and docs and can not find an answer.

**Comment by pascalkuthe** [comment]
You can select the first completion item with tab. Pre-election is not possible since selecting automatically means accepting a completion if you keep typing.

**Reply by xzbdmw** [reply]
New to helix, and figure it out that select an item indeed means you already type enter in other editors, which is kinda strange.
